handlers.js

```diff
@@ -79,6 +79,22 @@ async function rank(c, player, num) {
     }
 }
 
+function reply_wait(f)
+{
```

Comment:
Can you put this on the same line?

Reply:
Resolved in https://github.com/arispoloway/tempus-discord-bot/pull/2/commits/63778ca1eccd7944977fd2ed403f5471db2dd379